umbraco-cloud/set-up/external-login-providers.md

```diff
@@ -180,7 +180,16 @@ The **alias** must be unique across different login providers in the same enviro
+</td><td>A known case for using a special metadata address is when using Entra Id, and you have set up Claims mapping. In this case, you need to set the metadata address to the following:
+<code>https://login.microsoftonline.com/{tenant}/v2.0/.well-known/openid-configuration?appid={client-id}</code></td></tr></tbody></table>
+
+### Handling invites when using an External Login Provider
+
+When using an External Login Provider, we are no longer able to handle the invitation flow to the backoffice. This is because the user needs to be created in the External Login Provider in order to log in. Umbraco Cloud can not handle this integration. 
+So when using an External Login Provider, it is up to you as an administrator to handle how users get access to the backoffice. 
```

Comment:
```suggestion
When using an External Login Provider, the invitation flow to the backoffice can no longer be managed within Umbraco. This is because users must first be created in the External Login Provider before they can log in. Umbraco Cloud does not handle this integration.

As an administrator, you are responsible for managing user access to the backoffice.
```